be/src/olap/row.h

```diff
@@ -142,7 +142,7 @@ template<typename RowType>
 void agg_finalize_row(const std::vector<uint32_t>& ids, RowType* row) {
     for (uint32_t id : ids) {
         auto cell = row->cell(id);
-        row->schema()->column(id)->agg_finalize(&cell);
+        row->schema()->column(id)->agg_finalize(&cell, row->arena());
```

Comment:
If we use row's arena to do finalize. I'm afraid this arena will occupy a lot of memory if it is finalized many times.

Reply:
update and will release the memory immediately